vaadin-grid-flow-parent/vaadin-grid-testbench/src/main/java/com/vaadin/flow/component/grid/testbench/GridElement.java

```diff
@@ -54,14 +54,9 @@ public void scrollToRow(int row) {
         waitUntilLoadingFinished();
     }
 
-    /**
-     * Scrolls to the row with the given flat row index.
-     *
-     * @param row
-     *            the row to scroll to
-     */
-    protected void scrollToFlatRow(int row) {
```

Comment:
Should we still deprecate and keep this?

Reply:
I don't think so. It might be quite confusing otherwise. I think documenting it in breaking changes should be fine.